lib/addons/calico/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ClusterAddOn, ClusterInfo } from "../../eksBlueprintStack";
+
+export class CalicoAddon implements ClusterAddOn {
+
+    deploy(clusterInfo: ClusterInfo): void {
```

Comment:
was it validated on EKS? If yes, which cluster did you create?

Reply:
I validated your branch by creating a cluster so all good!
We need to agree on the validation process just in case: submitted PR if includes code change are verified by the submitter (and reviewer). 

I do see some extra junk that calico helm chart through like typha horizontal autoscaler. We need to check if it is really needed. May be the chart has an option to exclude it. Looks like it is needed for BPF and Felix: https://docs.projectcalico.org/reference/typha/